charts/rancher-eks-operator/1.0.800/Chart.yaml

```diff
@@ -5,4 +5,4 @@ home: https://github.com/rancher/eks-operator
 sources:
   - "https://github.com/rancher/eks-operator"
 version: 1.0.800
-appVersion: v1.0.8-rc2
+appVersion: v1.0.8
```

Comment:
This is ok, since this is what was already in 2.5 release branch. We should ensure the changes here are only differences between those two.